#!/bin/sh /usr/share/dpatch/dpatch-run
## 09-quiet-grep.dpatch by Michael Meskes <meskes@debian.org>
## modified by Patrick Winnert <winnie@debian.org>
##
## DP: Make grep be quiet
## DP: Add an short description to make lintian silent.

@DPATCH@

diff -urNad virtualbox-ose-1.5.2-dfsg2~/src/VBox/Installer/linux/vboxnet.sh virtualbox-ose-1.5.2-dfsg2/src/VBox/Installer/linux/vboxnet.sh
--- virtualbox-ose-1.5.2-dfsg2~/src/VBox/Installer/linux/vboxnet.sh	2007-12-16 12:43:28.000000000 +0100
+++ virtualbox-ose-1.5.2-dfsg2/src/VBox/Installer/linux/vboxnet.sh	2007-12-16 12:44:51.000000000 +0100
@@ -23,7 +23,8 @@
 # Required-Stop:
 # Default-Start:  3 5
 # Default-Stop:
-# Description:    VirtualBox permanent host networking setup
+# Short-Description: Virtualbox permanent host networking setup
+# Description:  This initscript sets up the networking for Virtualbox
 ### END INIT INFO
 
 PATH=$PATH:/bin:/sbin:/usr/sbin
@@ -236,7 +237,7 @@
     done < "$CONFIG"
     # Set /dev/net/tun to belong to the group vboxusers if it exists and does
     # yet belong to a group.
-    if ls -g "$TAPDEV" 2>/dev/null | grep root
+    if ls -g "$TAPDEV" 2>/dev/null | grep -q root
     then
       chgrp vboxusers "$TAPDEV"
       chmod 0660 "$TAPDEV"
